Documentation/Books/Manual/Views/ArangoSearch/DetailedOverview.md

```diff
@@ -75,118 +72,139 @@ During view modification the following directives apply:
 
 ## View properties
 The following terminology from ArangoSearch architecture is used to understand
-view properties assignment of its type:<br/>
-The index consists of several independent segments and the index **segment** itself
-is meant to be treated as a standalone index. **Commit** is meant to be treated
-as the procedure of accumulating processed data
-creating new index segments. **Consolidation** is meant to be treated as the procedure
-of joining multiple index segments into a bigger one and removing garbage documents
-(e.g. deleted from a collection). **Cleanup** is meant to be treated as the
-procedure of removing unused segments after release of internal resources.
-
-* **cleanupIntervalStep** (_optional_; type: `integer`; default: `10`; to disable use: `0`)<br/>
+view properties assignment of its type:
+
+The index consists of several independent segments and the index **segment**
```

Comment:
Should we have a glossary related to ArangoSearch?

Reply:
it seems we have some content here - to be double checked if it is old text or not: https://docs.arangodb.com/devel/Manual/Appendix/Glossary.html#view

Reply:
Will not do it right now, because it requires change of a structure.